Nesteo.Server/Services/Implementations/InspectionService.cs

```diff
@@ -27,5 +27,10 @@ public Task<InspectionPreview> FindPreviewByIdAsync(int id, CancellationToken ca
         {
             return Entities.Where(entity => entity.Id.Equals(id)).ProjectTo<InspectionPreview>(Mapper.ConfigurationProvider).FirstOrDefaultAsync(cancellationToken);
         }
+
+        public IAsyncEnumerable<Inspection> FindByBoxIdAsync(string id)
+        {
+           return Entities.Where(entity => entity.NestingBox.Id.Equals(id)).ProjectTo<Inspection>(Mapper.ConfigurationProvider).AsAsyncEnumerable();
```

Comment:
In C# you don't have to write `Equals()` as in Java. You can always use `==` instead because the `==` operator in C# is way more powerful and more dynamic than the one in Java.